utils/githooks/pre-commit.d/30-yamllint.sh

```diff
@@ -14,4 +15,13 @@ then
 fi
 
 echo "Checking yaml formatting"
-yamllint --strict .
+targets=(
+    '*.yml'
+    '*.yaml'
+)
+
+files=${files:=$(git diff --diff-filter=ACMRTUXB --name-only --cached -- "${targets[@]}")}
```

Comment:
As an aside,

```suggestion
files=${files:-$(git diff --diff-filter=ACMRTUXB --name-only --cached -- "${targets[@]}")}
```
Is usually used in this kind of construct.  `:-` simply expands to the LHS if it's set or the RHS if not which is then assigned to `$files`.  `:=` assigns the RHS to the LHS if it's not set so you are effectively double assigning `$files` in that case.